common/lib/wrapper_property.ts

```diff
@@ -71,12 +71,27 @@ export class WrapperProperties {
     null
   );
 
+  /**
+   * @deprecated since version 1.0.1 and replaced by wrapper property WRAPPER_QUERY_TIMEOUT.
```

Comment:
Should be 1.1.0.
Bumping to 1.1.0 because configuration profiles is a feature release.